app/views/refinery/authentication/devise/admin/users/_user.html.erb

```diff
@@ -1,3 +1,13 @@
+<% # setup params for various action links
+  edit_url =  refinery.edit_authentication_devise_admin_user_path(user)
+  delete_url = refinery.authentication_devise_admin_user_path(user)
```

Comment:
can you use `||=` for this and `edit_url` too please?

Reply:
done :)